ci/docker_sources/hil_container/runhil.sh

```diff
@@ -0,0 +1,4 @@
+nohup gunicorn -w 5 -b 0.0.0.0:7000 wsgi:application >/dev/null 2>&1 &
+sleep 1
+nohup hil-admin serve_networks >/dev/null 2>&1 &
```

Comment:
the command is `hil-admin serve-networks` we have dashes everywhere instead of an underscore.

Reply:
huh, makes me wonder why this even works then

Reply:
I think for some reason upon exiting the bash script the two servers get killed.  I tried to do a `kill %1 %2` but it said there was nothing running

Reply:
I don't think it works. 
```
[centos@hil etc]$ hil-admin serve_networks > /dev/null 2>&1 &
[1] 27435
[centos@hil etc]$ 
[centos@hil etc]$ 
[centos@hil etc]$ 
[1]+  Exit 2                  hil-admin serve_networks > /dev/null 2>&1

```
Since we push the output to /dev/null we don't see anything on screen. and it exits in a few seconds.

Reply:
I'll take out the gunicorn bit and see if the server is still running

Reply:
There must be some magic going on here because if I take out the gunicorn server I can't curl to hil at all

Reply:
Sorry if I wasn't clear. The gunicorn server is the api sever and that should stay. Without it, you won't be able to curl to hil at all (unless you are running the dev server using the hil-admin command).

The serve-networks command is the process that does the networking operations, if it's not running and you queue a networking operation, that operation wouldn't go through. You would still be able to curl to hil and do other things. so the bit that should get removed is `hil-admin serve_networks`... since that is a command which doesn't exist. And you are running the right command in `setupdb.sh` already.

Reply:
Right I get that, but I believe the two server commands I run in that setupdb script get killed somewhere along the way, which is why it's fine to first run them there, setup the db, then run the servers again at the end.  Also I fixed the serve-networks command to be correct in the latest commit